data/recommendation/notify.js

```diff
@@ -100,8 +100,8 @@ class Notify {
   createWarningFooter() {
     return yo`
       <footer>
-        <div onclick=${this.handlePasswordChange}>I've changed my password</div>
-        <div onclick=${this.handleNoAccount}>I don't have an account</div>
+        <div onclick=${this.handlePasswordChange}>I don't have an account</div>
```

Comment:
Actually they were backwards before (compared to the mockup), and now they match the mockup. It seems to me that convention is  to have the "positive" button on the right, and the current layout matches it. 

Do you think that we should have "thanks!" on the right or left?